IaC/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "db_username" {
+  
+}
+
+variable "db_password" {
+  
+}
+
+variable "function_name" {
```

Comment:
You can have a more general `api_name`, and compute others (`functon_name`, `role_name`, `policy_name` etc) based on it, so that you define less variables on the module.